docs/content/docs/2.guide/3.custom-paths.md

```diff
@@ -216,3 +216,20 @@ To configure a custom path for a dynamic route, you need to use it in double squ
 ::callout{type="info"}
 `defineI18nRoute` compiler macro is tree-shaked out at build time and is not included in the dist files.
 ::
+
+
+### `definePageMeta({ name: '...' })` caveat
+
+By default Nuxt overwrites generated route values at build time which breaks custom named routes (setting `name` with `definePageMeta`) when resolving localized paths.
+
+Nuxt v3.10 introduced the experimental feature [`scanPageMeta`](https://nuxt.com/docs/guide/going-further/experimental-features#scanpagemeta), this needs to be enabled for custom named routes to work when using Nuxt I18n.
+
+This experimental feature can be enabled as shown here:
```

Comment:
I think it might suggest using `::callout{type="warning"}` to emphasize that this is still an experimental feature.
the developer may be used as a STABLE feature. :)